src/ui/public/styles/base.less

```diff
@@ -289,28 +289,108 @@ table {
 
 //== SavedObjectFinder
 saved-object-finder {
+  .row {
+    background-color: @kibanaGray6;
+    padding: 10px;
+    display: flex;
+    flex-direction: row;
+  }
+
   .form-group {
     margin-bottom: 0;
+    float: left;
+    flex: 8;
+
     input {
-      .border-bottom-radius(0);
+      border: none;
+      padding: 5px 0px;
+      border-radius: @border-radius-base;
+      text-transform: capitalize;
     }
-  }
 
-  .list-group-item {
-    a {
+    span {
+      padding: 5px 0px;
+      background-color: @white;
+      border: none;
+
       i {
-        color: @saved-object-finder-icon-color !important;
+        color: @kibanaGray3;
+        width: 15px;
       }
+    }
+  }
+
+  .finder-hit-count, .finder-manage-object {
```

Comment:
Put this under the saved object finder above. Same with all of the .finder* classes